### PR TITLE
Rephrased for clarity.

### PR DESCRIPTION
**Pull request by patrikbego**

### CODE REVIEW

**Comment by iliakan** [comment]
Thanks. I'm not sure though, why "Set-like" is bad?

**Comment by iliakan** [comment]
People say "Array-like", and that's normal.

File: 1-js/05-data-types/08-weakmap-weakset/article.md [hunk 1]
```diff
@@ -248,7 +248,7 @@ obj = null;
 - An object exists in the set while it is reachable from somewhere else.
 - Like `Set`, it supports `add`, `has` and `delete`, but not `size`, `keys()` and no iterations.
 
-Being "weak", it also serves as an additional storage. But not for an arbitrary data, but rather for "yes/no" facts. A membership in `WeakSet` may mean something about the object.
+Being "weak", it also serves as additional storage. But not for arbitrary data, but rather for "yes/no" facts. A membership in `WeakSet` may mean something about the object.
```
Comment (iliakan) [inline 1-js/05-data-types/08-weakmap-weakset/article.md:251]: Please, remove one "But" here too =)

**Review by iliakan — CHANGES_REQUESTED** [review]
Please, remove one "But" where commented, and - are you sure that "set-like" is bad?

File: 1-js/05-data-types/08-weakmap-weakset/article.md (hunk 1 above)
Comment (patrikbego) [reply 1-js/05-data-types/08-weakmap-weakset/article.md]: Yes, that is better ;)

**Comment by patrikbego** [comment]
> Please, remove one "But" where commented, and - are you sure that "set-like" is bad?
You are right, it actually is dramatically correct. I put it back.

**Comment by iliakan** [comment]
Thanks!

**Pull request merged**